{"links": {"self": {"href": "data/repositories/osrf/gazebo/pullrequests/1506/comments/6081151.json"}, "html": {"href": "#!/osrf/gazebo/pull-requests/1506/_/diff#comment-6081151"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 1506, "links": {"self": {"href": "data/repositories/osrf/gazebo/pullrequests/1506.json"}, "html": {"href": "#!/osrf/gazebo/pull-requests/1506"}}, "title": "Apply Force/Torque GUI"}, "content": {"raw": "New crash (or just new backtrace). Happened when closing the dialog.\n\n~~~\nQt has caught an exception thrown from an event handler. Throwing\nexceptions from an event handler is not supported in Qt. You must\nreimplement QApplication::notify() and catch all exceptions there.\n\nterminate called after throwing an instance of 'std::bad_alloc'\n  what():  std::bad_alloc\n\nProgram received signal SIGABRT, Aborted.\n0x00007ffff6c20cc9 in __GI_raise (sig=sig@entry=6) at ../nptl/sysdeps/unix/sysv/linux/raise.c:56\n56\t../nptl/sysdeps/unix/sysv/linux/raise.c: No such file or directory.\n(gdb) bt\n#0  0x00007ffff6c20cc9 in __GI_raise (sig=sig@entry=6) at ../nptl/sysdeps/unix/sysv/linux/raise.c:56\n#1  0x00007ffff6c240d8 in __GI_abort () at abort.c:89\n#2  0x00007ffff72266b5 in __gnu_cxx::__verbose_terminate_handler() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#3  0x00007ffff7224836 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#4  0x00007ffff7224863 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#5  0x00007ffff7224af6 in __cxa_rethrow () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#6  0x00007ffff3c9a596 in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) () from /usr/lib/x86_64-linux-gnu/libQtCore.so.4\n#7  0x00007ffff3c9fb79 in QCoreApplication::exec() () from /usr/lib/x86_64-linux-gnu/libQtCore.so.4\n#8  0x00007ffff778519b in gazebo::gui::run (_argc=1, _argv=0x7fffffffe008) at /home/louisep/gazebo_source/gazebo/gazebo/gui/GuiIface.cc:292\n#9  0x00000000004010d9 in main (_argc=<optimized out>, _argv=<optimized out>) at /home/louisep/gazebo_source/gazebo/gazebo/gui/main.cc:23\n~~~", "markup": "markdown", "html": "<p>New crash (or just new backtrace). Happened when closing the dialog.</p>\n<div class=\"codehilite\"><pre><span></span>Qt has caught an exception thrown from an event handler. Throwing\nexceptions from an event handler is not supported in Qt. You must\nreimplement QApplication::notify() and catch all exceptions there.\n\nterminate called after throwing an instance of &#39;std::bad_alloc&#39;\n  what():  std::bad_alloc\n\nProgram received signal SIGABRT, Aborted.\n0x00007ffff6c20cc9 in __GI_raise (sig=sig@entry=6) at ../nptl/sysdeps/unix/sysv/linux/raise.c:56\n56  ../nptl/sysdeps/unix/sysv/linux/raise.c: No such file or directory.\n(gdb) bt\n#0  0x00007ffff6c20cc9 in __GI_raise (sig=sig@entry=6) at ../nptl/sysdeps/unix/sysv/linux/raise.c:56\n#1  0x00007ffff6c240d8 in __GI_abort () at abort.c:89\n#2  0x00007ffff72266b5 in __gnu_cxx::__verbose_terminate_handler() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#3  0x00007ffff7224836 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#4  0x00007ffff7224863 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#5  0x00007ffff7224af6 in __cxa_rethrow () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6\n#6  0x00007ffff3c9a596 in QEventLoop::exec(QFlags&lt;QEventLoop::ProcessEventsFlag&gt;) () from /usr/lib/x86_64-linux-gnu/libQtCore.so.4\n#7  0x00007ffff3c9fb79 in QCoreApplication::exec() () from /usr/lib/x86_64-linux-gnu/libQtCore.so.4\n#8  0x00007ffff778519b in gazebo::gui::run (_argc=1, _argv=0x7fffffffe008) at /home/louisep/gazebo_source/gazebo/gazebo/gui/GuiIface.cc:292\n#9  0x00000000004010d9 in main (_argc=&lt;optimized out&gt;, _argv=&lt;optimized out&gt;) at /home/louisep/gazebo_source/gazebo/gazebo/gui/main.cc:23\n</pre></div>", "type": "rendered"}, "created_on": "2015-04-06T17:21:06.637199+00:00", "user": {"display_name": "Louise Poubel", "uuid": "{5cfa2075-477b-4ded-bdb9-8d2479544ec4}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D"}, "html": {"href": "https://bitbucket.org/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8/7d903d90-c5ea-4182-b7ef-0d467e9e1c74/128"}}, "nickname": "chapulina", "type": "user", "account_id": "557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8"}, "updated_on": "2015-04-06T17:21:06.644369+00:00", "type": "pullrequest_comment", "id": 6081151}